docs/docusaurus/docs/deployment_patterns/how_to_instantiate_a_data_context_on_an_emr_spark_cluster.md

```diff
@@ -2,47 +2,39 @@
 title: How to instantiate a Data Context on an EMR Spark cluster
 ---
 import Prerequisites from './components/deployment_pattern_prerequisites.jsx'
```

Comment:
@Rachel-Reverie I'll leave this for now. I'm juggling multiple priorities, and I don't have the bandwidth to implement this fix currently. It would take me significant time to determine how you've been calling the correct prerequisites.